packages/runtime/container-runtime/src/test/blobManager.spec.ts

```diff
@@ -384,6 +415,46 @@ describe("BlobManager", () => {
 		assert.strictEqual(summaryData.redirectTable.size, 1);
 	});
 
+	it("throw if expired", async () => {
+		let exceptionOccurred = false;
+		await runtime.attach();
+		runtime.useTTLStorage = true;
+		const handle = runtime.createBlob(IsoBuffer.from("blob", "utf8"));
+		await runtime.processBlobs();
+		await handle;
+
+		await runtime.connect(50).catch((error) => {
+			assert.strictEqual(
+				error.message,
+				"Trying to send BlobAttachOp of expired blob",
+				"Unexpected exception thrown",
+			);
+			exceptionOccurred = true;
+		});
+		assert.strictEqual(
+			exceptionOccurred,
+			true,
+			"call did not fail as expected.",
+		);
+		await runtime.processAll();
+	});
+
+	it("no reupload if not expired", async () => {
```

Comment:
Does this test verify that we don't reupload?

Reply:
This was a test that I used to verify we reuploaded when expired but removing it since we will assert